### ISSUE TITLE
Add support for proxy for Ubuntu builds

### ISSUE THREAD

**Issue by rismoney** [vmware-samples/packer-examples-for-vsphere#228]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.

### Description

The ubuntu images subiquity template (have only tested 22.04)
The following should be added to support a web proxy.

```yaml
  refresh-installer:
    update: yes
```
^ this should be added to fix autoinstall bugs in general.  The beta is not needed.

There is a bug that breaks this command from properly functioning:
```yaml
  proxy: http://myproxy:port
```
This statement would be optional unless a user specified a proxyurl variable.

### Use Case(s)

Build systems without direct internet access

### Potential Configuration

see above

### References

https://bugs.launchpad.net/subiquity/+bug/1970409/comments/13

**Comment by tenthirtyam** [comment]
Hi @rismoney, 👋 

Please update the description. 

For example: "Add support for <x>."

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.

**Comment by tenthirtyam** [comment]
Closed in favor of #232 and this should cover all OSes.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.
